2-mandatory/6-writing-tests-advanced.js

```diff
@@ -2,7 +2,7 @@
   For this exercise, the function has been written for you and you need to write
   the tests.
 
-  This function takes a trainee parameter that is an object. The object
```

Comment:
Tests are all ok but read this again and check the function convertScoreToGrade() again.